rosie/core/__init__.py

```diff
@@ -28,7 +28,10 @@ def __init__(self, settings, adapter):
         self.settings = settings
         self.dataset = adapter.dataset
         self.data_path = adapter.path
-        self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        if self.settings.UNIQUE_IDS:
+            self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        else:
+            self.suspicions = self.dataset.copy()
```

Comment:
`document_id` will never be a great combination for `UNIQUE_IDS` because there is some receipts that don't have one, or some receipts that have `sem fatura` problem :/